package.json

```diff
@@ -51,6 +51,7 @@
     "rollup": "^2.7.5",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "uglify-js": "^3.9.4",
```

Comment:
Uglify is needed to run `npm run build`. Serban and I must have had this installed globally on our machines previously.